src/main/java/uk/ac/cam/cl/dtg/segue/dos/content/Content.java

```diff
@@ -202,6 +205,18 @@ public final void setChildren(List<ContentBase> children) {
         this.children = children;
     }
 
+    public Boolean getDeprecated() {
+        if (null == deprecated) {
```

Comment:
Do we need this check? I think it would be nicer if we had a front end undefined check so that this isn't nescessary?

Reply:
In fact, the front end PR does have an undefined check :)

Reply:
I was copying `getPublished` for consistency, but to be fair it really doesn't need to be there (never a good idea to copy bad code anyway) - I've removed it 👍